samples/sample-junit5/build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 plugins {
     java
     jacoco
-    id("org.gradle.test-retry") version "1.0.0"
+    //TODO this needs to replaced as soon as 1.1.7 is out together with junit5.sample.out#L9 and remove `--warning-mode all` from junit5.sample.confL2
```

Comment:
Rather than a TODO comment, should we rather create an issue so we won't forget?

Reply:
I propose to change the sample build so that we can test with multiple versions and inject the version via the `GradleRunner` as you suggested and then we do not need the TODO anymore.

Reply:
I mixed up `GradleSamplesRunner` with `GradleRunner` in your comment. I could get rid of the TODO by applying some templating to fetch the latest plugin version.